documentation/airport-guides/lszh.md

```diff
@@ -0,0 +1,61 @@
+### About this facility
+Skyguide Air Navigation Center, Dübendorf, Switzerland
+269,160 annual operations in 2016
+
+Located at Dübendorf Air Base (close to Zurich), besides the LSAZ FIR, the Skyguide Air Navigation Center is also responsible for the in- and outbound traffic of Switzerlands major airport Zurich (LSZH).
+
+### Airspace
+The airspace encompasses a large area around Zurich up to FL190 and also includes some German territory. Since approaches to Runways 14 and 16 require the use of German airspace, these approaches aren't available during the German curfew period (MON - FRI: 2100-0700LT; SAT, SUN, holidays: 2000-0700LT). During this period Runways 28 or 34 are usually used which reduces the capacity of the airport.
+
+### Instrument Procedures
+LSZH offers several departure procedures, however the DEGES and VEBIT departures are the most commonly used. Arrivals follow their STAR's to either GIPOL or AMIKI, where aircraft must be vectored to their final approach course.
+
+### Sectors Available in OpenScope
+```
+D - Zurich Departure
+W - Zurich Arrival (West)
+E - Zurich Arrival (East)
+F - Zurich Final Approach
+```
+
+### Airports of Interest
+
+#### LSZH (Primary Airport)
+Zurich airport                               
+269,160 annual operations (2016)                               
+
+Zürich Airport, also known as Kloten Airport, is the largest international airport of Switzerland and the principal hub of Swiss International Air Lines. It serves Zürich, Switzerland's largest city, and, with its surface transport links, much of the rest of the country. The airport is located 13 kilometres (8 mi) north of central Zürich, in the municipalities of Kloten, Rümlang, Oberglatt, Winkel and Opfikon, all of which are within the canton of Zürich. The preferred runway setup is runway 14 for landings and runway 28 for departures. Heavy aircraft may take runway 16 for departing due to the performance reasons but departures out of runway 28 have priority. Runway 10 is only going to be used if meteorological reasons require.
+
+#### LSZR
+Flugplatz St. Gallen-Altenrhein (St. Gallen–Altenrhein Airport)                               
+26,382 annual operations (2016)                               
+40nm to the east of LSZH                               
+
+St. Gallen–Altenrhein Airport is a small airport in Altenrhein in the Canton of St. Gallen, Switzerland near Lake Constance. It is also known under its trading name People's Business Airport and is the home base for People's Viennaline.
```

Comment:
We should mention that it is controlled by the ARFA sector up to FL100 (I think?)